### PR TITLE
chore(guide): Changed discord shield to button

### PR DESCRIPTION
**Pull request by TRaSH-**
# Pull Request

## Purpose

- Changed discords shields.io to a click here button, hopefully to solve the rate limiting issue.
- Changed `{! include-markdown "../../../includes/support.md" !}` to `--8<-- "includes/support.md"` to be more consistent with the other pages and less failures depending on where the page is located 

## Approach

- [x] Changed discords shields.io to a click here button
- [x] Changed `{! include-markdown "../../../includes/support.md" !}` to `--8<-- "includes/support.md"`

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by zakkarry** [comment]
Not that it matters to me, but if you're using cloudflare for this, couldn't you just cache the asset and set a TTL on the image so it updates only every 5m or something rather than just removing it?

I'm not sure what rate limiting issue really is, or what its cause, but it seems like something easily fixed without entirely removing it?

@bakerboy448

**Comment by bakerboy448** [comment]
Zak's triage seems likely accurate

**Comment by TRaSH-** [comment]
I only removed it from the guide pages and kept it on the main page (index) where it also shows the other badges.

I wouldn't know if I can cache the shields, used to work always without issues. The others do, except the discord one.

File: docs/Plex/Tips/Plex-media-server.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ All changes need to be done on your Plex Media Server.
 
 !!! info "Why didn't you cover option .... ?"
 
-    I will only cover settings that I think might be interesting or are personal preference, if you would like me to cover some other settings, please contact me on [![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord)](https://trash-guides.info/discord){:target="\_blank" rel="noopener noreferrer"}
+    We will only cover settings that we think might be interesting or are personal preference, if you would like me to cover some other settings, please contact us on discord.
```
Comment (zakkarry) [inline docs/Plex/Tips/Plex-media-server.md:21]:
```suggestion
    We will only cover settings that we think might be interesting or are personal preference, if you would like me to cover some other settings, please contact me on [![Discord chat](https://trash-guides.info/assets/discord.png)](https://trash-guides.info/discord){:target="\_blank" rel="noopener noreferrer"}

File: docs/Radarr/radarr-setup-quality-profiles.md [hunk 2]
```diff
@@ -77,9 +77,9 @@ At the bottom, in your chosen profile, you will see the added Custom Formats whe
 
     ![Flowchart](/Radarr/images/flowchart-quality-profiles-radarr.png)
 
-    If you're unsure or have questions do not hesitate to ask for help on Discord
+    <center>If you're unsure or have questions do not hesitate to ask for help on Discord</center>
 
-    [![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}
+    <center>[ Click For Support ](https://trash-guides.info/discord){ .md-button .md-button--primary }</center>
```
Comment (zakkarry) [inline docs/Radarr/radarr-setup-quality-profiles.md:82]:
```suggestion
    [![Discord chat](https://trash-guides.info/assets/discord.png){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}

File: docs/Sonarr/sonarr-setup-quality-profiles.md [hunk 3]
```diff
@@ -72,9 +72,9 @@ At the bottom, in your chosen profile, you will see the added Custom Formats whe
 
     ![Flowchart](/Sonarr/images/flowchart-quality-profiles-sonarr.png)
 
-    If you're unsure or have questions do not hesitate to ask for help on Discord
+    <center>If you're unsure or have questions do not hesitate to ask for help on Discord</center>
 
-    [![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}
+    <center>[ Click For Support ](https://trash-guides.info/discord){ .md-button .md-button--primary }</center>
```
Comment (zakkarry) [inline docs/Sonarr/sonarr-setup-quality-profiles.md:77]:
```suggestion
    [![Discord chat](https://trash-guides.info/assets/discord.png){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}

File: includes/cf/score-attention.md [hunk 4]
```diff
@@ -6,4 +7,5 @@
 
     If you're unsure or have questions do not hesitate to ask for help on Discord
 
-    [![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}
+    <center>[ Click For Support ](https://trash-guides.info/discord){ .md-button .md-button--primary }</center>
```
Comment (zakkarry) [inline includes/cf/score-attention.md:10]:
```suggestion
    [![Discord chat](https://trash-guides.info/assets/discord.png){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}

File: includes/support.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 !!! question "Questions or Suggestions?"
 
     <center>If you have questions or suggestions, click the button below to join our Discord server.</center>
 
     <center>[ Click For Support ](https://trash-guides.info/discord){ .md-button .md-button--primary }</center>
-
-    <center>[![Discord](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}</center>
+<!-- markdownlint-enable MD041-->
```
Comment (zakkarry) [inline includes/support.md:9]:
```suggestion

    [![Discord chat](https://trash-guides.info/assets/discord.png){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}
<!-- markdownlint-enable MD041-->

File: docs/index.md [hunk 6]
```diff
@@ -25,10 +25,11 @@ I try to make my guides as easy as possible for everyone to understand and, in m
...
 ## Support
 
-[![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="\_blank" rel="noopener noreferrer"}
+- If you have questions or suggestions, click the button below to join our Discord server.
+- If you want to contribute, feel free to create a PR (Pull Request).
+- If you found a actual issue related to the guide you can open a Github [issue](https://github.com/TRaSH-Guides/Guides/issues){:target="\_blank" rel="noopener noreferrer"}.
 
-If you have questions, suggestions, or requests, or want to contribute, click the chat badge to join the Discord Channel. There you can ask your questions directly, or you can use the following [link](https://github.com/TRaSH-Guides/Guides/issues){:target="\_blank" rel="noopener noreferrer"} to open an issue on GitHub.
-Feel free to create a PR if you want to.
+<center>[ Click For Support ](https://trash-guides.info/discord){ .md-button .md-button--primary }</center>
```
Comment (zakkarry) [inline docs/index.md:32]:
```suggestion
- If you have questions or suggestions, click the button below to join our Discord server.
- If you want to contribute, feel free to create a PR (Pull Request).
- If you found a actual issue related to the guide you can open a Github [issue](https://github.com/TRaSH-Guides/Guides/issues){:target="\_blank" rel="noopener noreferrer"}.

[![Discord chat](https://trash-guides.info/assets/discord.png){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}

**Review by zakkarry — CHANGES_REQUESTED** [review]
cached image on cloudflare

**Pull request merged**